lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -352,23 +373,19 @@ def insert_at_position(position)
             if in_list?
               old_position = send(position_column).to_i
               return if position == old_position
-              shuffle_positions_on_intermediate_items(old_position, position)
+              # temporary move after bottom with gap, avoiding duplicate values
+              # gap is required to leave room for position increments
+              # positive number will be valid with unique not null check (>= 0) db constraint
+              tmp_position_after_everything = acts_as_list_class.unscoped.maximum(position_column).to_i + 2
```

Comment:
Does this unscoping remove the `acts_as_list` scope? I assume you're not wanting to the position column to be globally unique? just unique per scope?